NOTES-NONSTOP.md

```diff
@@ -0,0 +1,148 @@
+ 
+ * `nonstop-nsx` or default will select an unthreaded build.
+ * `nonstop-nsx_put` selects the PUT build.
+ * `nonstop-nsx_64_put` selects the 64 bit file length PUT build.
+ * `nonstop-nsx_spt_floss` selects the SPT build with FLOSS. FLOSS is
+   required for SPT builds because of a known hang when using SPT on its own.
+ 
+### TNS/E Considerations
+   
+ The TNS/E platform is build using the same set of builds specifying `nse`
+ instead of `nsx` in the set above.
+ 
+ You cannot build for TNS/E for FIPS, so you must specify the `no-fips`
+ option to `./Configure`
+ 
+About Prefix and OpenSSLDir
+---------------------------
+
+ Because there are many potential builds that must co-exist on any given
+ NonStop node, managing the location of your build distribution is crucial.
+ Keep each destination separate and distinct. Mixing any mode described in
+ this document can cause application instability. The recommended approach
+ is to specify the OpenSSL version and threading model in your configuration
+ options, and keeping your memory and float options consistent, for example:
+ 
+ * For 1.1 `--prefix=/usr/local-ssl1.1 --openssldir=/usr/local-ssl1.1/ssl` 
+ * For 1.1 PUT `--prefix=/usr/local-ssl1.1_put --openssldir=/usr/local-ssl1.1_put/ssl`
+ * For 3.0 `--prefix=/usr/local-ssl3.0 --openssldir=/usr/local-ssl3.0/ssl` 
+ * For 3.0 PUT `--prefix=/usr/local-ssl3.0_put --openssldir=/usr/local-ssl3.0_put/ssl`
+
+ Use the `_RLD_LIB_PATH` environment variable in OSS to select the appropriate
+ directory containing `libcrypto.so` and `libssl.so`. In GUARDIAN, use the
+ `=_RLD_LIB_PATH` search define to locate the GUARDIAN subvolume where OpenSSL
+ is installed.
+
+Float Considerations
+--------------------
+
+ OpenSSL is built using IEEE float mode by default. If you need a different
+ IEEE mode, create a new configuration specifying `tfloat-x86-64` (for Tandem
+ float) or `nfloat-x86-64` (for Neutral Float).
+ 
+Memory Models
+-------------
+
+ The current OpenSSL default memory model uses the default platform address
+ model. If you need a different address model, you must specify the appropriate
+ c99 options for compile (`CFLAGS`) and linkers (`LDFLAGS`).
+
+Cross Compiling with NSDEE
+--------------------------
+
+ **Note:** None of these builds have been tested by the platform maintainer and are
+ supplied for historical value. Please submit a Pull Request to OpenSSL should
+ these need to be adjusted.
+ 
+ If you are attempting to build OpenSSL with NSDEE, you will need to specify
+ the following variables. The following set of compiler defines are required:
+ 
+`   COMP_ROOT=$(cygpath -w /path/to/comp_rooot)   # must be path format for system (ie windows)`
+`   CC=/path/to/c99                               # must be executable by shell`
+ 
+### Optional Build Variables 
+
+`   DBGFLAG="--debug"`
+`   CIPHENABLES="enable-ssl3 enable-ssl3-method enable-weak-ssl-ciphers enable-rc4"`
+
+### Internal Known On Platform Cross Compile Variables
```

Comment:
What do you mean with `Internal Known On ...`?  Do you mean `Internally known ...`?

Reply:
I cleared up the header. Cross compiles are allowed from one NonStop variant (x86) to another (ia64), in that direction.